chart/templates/extra-objects.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.extraObjects }}
+{{- range .Values.extraObjects }}
+---
+{{- tpl (toYaml .) $ | nindent 0 }}
```

Comment:
tweaked the config to use tpl. That way users can use helm dynamic fields in the extraObjects.